### PR TITLE
Removed instances or preview

### PR DESCRIPTION
**Pull request by molebox**
This PR removes all instances of `--preview-feature` and `Preview` related to `db push`

### CODE REVIEW

File: content/300-guides/050-database/400-prototyping-schema-db-push.mdx [hunk 1]
```diff
@@ -102,13 +102,13 @@ The following scenario demonstrates how to use `db push` to synchronize a new sc
     /* ... */
     ```
 
-5. Push the changes:
+5.  Push the changes:
```
Comment (janpio) [inline content/300-guides/050-database/400-prototyping-schema-db-push.mdx:105]: Nitpick: Why two spaces here?

Comment (janpio) [reply content/300-guides/050-database/400-prototyping-schema-db-push.mdx:105]: (Across the whole file here really)

File: content/300-guides/050-database/400-prototyping-schema-db-push.mdx [hunk 2]
```diff
@@ -152,71 +152,71 @@ The following scenario demonstrates how to use `db push` to prototype a change t
...
+       categories Category[]
+   }
+
+   model Category {
+       id    Int    @id @default(autoincrement())
+       name  String @db.VarChar(50)
+       posts Post[]
+       @@unique([name])
+   }
+   ```
```
Comment (janpio) [inline content/300-guides/050-database/400-prototyping-schema-db-push.mdx:196]: Sure the indentation of the code block still works in the rendered view, that it is _inside_ the list vs on the same level?

File: content/400-reference/200-api-reference/200-command-reference.mdx [hunk 3]
```diff
@@ -563,13 +563,13 @@ Validation Error Count: 1
 
 ## <inlinecode>db</inlinecode>
 
-> **Note**: `db push` and `db seed` are currently in [Preview](../../../about/releases#preview). Features in Preview may still have some issues, and are not recommended for use in production environments. To access commands in Preview, you must opt-in via the `--preview-feature` flag.
+> **Note**: `db seed` is currently in [Preview](../../../about/releases#preview). Features in Preview may still have some issues, and are not recommended for use in production environments. To access commands in Preview, you must opt-in via the `--preview-feature` flag.
```
Comment (janpio) [inline content/400-reference/200-api-reference/200-command-reference.mdx]: This note can probably be moved to `seed` then instead of being on this level?

**Review by janpio — APPROVED** [review]

File: content/300-guides/050-database/400-prototyping-schema-db-push.mdx (hunk 2 above)
Comment (molebox) [reply content/300-guides/050-database/400-prototyping-schema-db-push.mdx:196]: Man, these prettier formatting things are super annoying! On the plus side, the indentation doesnt affect its rendered output

File: content/300-guides/050-database/400-prototyping-schema-db-push.mdx (hunk 1 above)
Comment (molebox) [reply content/300-guides/050-database/400-prototyping-schema-db-push.mdx:105]: Prettier again....

File: content/400-reference/200-api-reference/200-command-reference.mdx (hunk 3 above)
Comment (molebox) [reply content/400-reference/200-api-reference/200-command-reference.mdx]: Good catch. Will move

File: content/300-guides/050-database/400-prototyping-schema-db-push.mdx (hunk 1 above)
Comment (janpio) [reply content/300-guides/050-database/400-prototyping-schema-db-push.mdx:105]: That would be an incorrect configuration then though for Markdown.

**Pull request merged**